docs/UserGuide.md

```diff
@@ -52,9 +52,9 @@ If you can type fast, TA<sup>2</sup> can get your contact and assignment managem
  * Result Message Box: Displays a success message if your command has been executed, or an error message if your command 
    has failed.
    
- * Contact List: Displays your student's relevant information. (Can be modified using `find`)
+ * Contact List Panel: Displays your student's relevant information. (Can be modified using `find`)
    
- * Assignment List: Displays a specified student's assignments. (Activated using `show`)
+ * Assignment List Panel: Displays a specified student's assignments. (Activated using `show`)
```

Comment:
For this, actually if we refer to student's "assignment list" as assignments, we can just omit panel. 

In fact, is it possible to standardise the naming convention of **assignment list** throughout the user guide (Ctrl + F)?
Currently, it has two meanings
1) the list of assignments a student has can be changed to (**assignments**).
2) the panel that stores assignments of students at the right side of the screen is the (**assignment list**)